### DISCUSSION TITLE
Test launches amount limitation for the project

### DISCUSSION THREAD

**Discussion by blick882** [allure-framework/allure2#1374] [TestOps]
Hello!
I have more than 22k test launches on one of the projects.
After pushing results of the new test run, it looks like this(but if you go to the tree, all looks fine):

![image](https://user-images.githubusercontent.com/48013818/128695220-3f078b8c-99b5-4a53-bf14-141fb1d2c3ae.png)

But if I push the same results to another project with fewer test runs, it looks fine.
This issue makes it not possible to use it in other integrations since API says that this test run is "null"

2 questions:
How to fix this and avoid this limitation?
How to set up the automatic cleanup of test runs of how to delete old test runs with one click?

Thank you!

**Comment by cheshi-mantu** [comment]
Hi, 
all the issues related to Allure TestOps must be submitted through different channel as per discussions policy (#1296). Please file your request registered with corporate email address here: https://support.qameta.io
see you there.

**Comment by baev** [comment]
Allure TestOps has no limits for results/launches count, so probably something went wrong. And in order to fix the problem we may need you to share some details on your installation, which is considered as private data. That's why please follow link to our private support and we'll try our best to help you
